stdlib/package.py

```diff
@@ -137,6 +139,176 @@ def __init__(
             shutil.rmtree(self.package_cache)
         os.makedirs(self.package_cache)
 
+    def is_empty(self) -> bool:
+        """Tests if the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
```

Comment:
Shouldn't this be an infinitive verb?
```suggestion
        """Test if the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
```

Reply:
It should, thanks!